### ISSUE TITLE
Accessibility: MAS40B: Inspect: Localized Control Type property is wrong for the "Trending" list Box

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#51]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open https://marketplace.visualstudio.com/VSCode in EDGE
2. Navigate to the "Trending" list box available under the "Most Popular" extensions 
3. Observe the "Localized Control Type" property in Inspect

**Actual Result:**
The "Localized Control Type" property is "Group"

**Narrator Behaviour:**
Narrator announces as "Group"

**Expected Result:**
The "Localized Control type" property should be "List Box"
Eg: Trending List Box

**User Impact:**
Visually impaired users will not be able to know that it is a list box as the Narrator announces as group.
Providing the correct control type property enables the compatibility with the Screen Reader, which may be used by people with disabilities.

**screenshots:**
![localized control type is group for trending list box](https://cloud.githubusercontent.com/assets/24825441/21718371/03352044-d43d-11e6-98c1-519d4d6c1876.png)

**Comment by waderyan** [comment]
Marketplace

**Comment by waderyan** [comment]
Please send to Marketplace team.
